services/offer.js

```diff
@@ -52,6 +52,18 @@ const offerService = {
     if (!offer) {
       throw createError(404, OFFER_NOT_FOUND)
     }
+  },
+
+  getPriceValues: async (authorRole) => {
+    const offers = await Offer.find().select('price -_id').lean().exec()
```

Comment:
in find we should filter by role, we need only tutors offers or students requests not all offens